### PR TITLE
Dutch translation file added.

### PR DESCRIPTION
**Pull request by mhartgring**
I like your project so I have created a Dutch translation file for you.

(I hope I did this right, it has been a while since I created a PR)

### CODE REVIEW

File: client/src/lang/translations/nl.json
Comment (Stokkie64) [inline client/src/lang/translations/nl.json:37]: Geschiedenis van de geselecteerde dag

File: client/src/lang/translations/nl.json
Comment (Stokkie64) [inline client/src/lang/translations/nl.json:46]: geïnstalleerde

File: client/src/lang/translations/nl.json
Comment (Stokkie64) [inline client/src/lang/translations/nl.json:56]: Back-up codes werken als eenmalige wachtwoorden. Je kunt ze altijd gebruiken in plaats van je gewone wachtwoord (bijvoorbeeld als je het bent vergeten), maar slechts één keer.

File: client/src/lang/translations/nl.json
Comment (Stokkie64) [inline client/src/lang/translations/nl.json:63]: gekopieerd

File: client/src/lang/translations/nl.json
Comment (Stokkie64) [inline client/src/lang/translations/nl.json:75]: Selecteer je (eerder geëxporteerde) ZIP-bestand om gegevens te uploaden. De teksten worden als nieuwste items behandeld. Importeren kan enkele minuten duren, afhankelijk van de bestandsgrootte.

**Review by Stokkie64 — CHANGES_REQUESTED** [review]
Thanks so much for your PR! It's really appreciated. 
I've added some small feedback suggestions. Mostly small things or subjective style points. I leave it up to you which you want to take with. :)

**Comment by mhartgring** [comment]
Good eye, I rushed into making this. I'll update the PR asap.

**Review by ArjenMarcoSingels — APPROVED** [review]
Akkoord, Happy with the language. I leave it to @PhiTux to merge this if he's happy :)

**Review by Stokkie64 — APPROVED** [review]
Approved, @PhiTux I approved with my incorrect github account. Are you able to remove that approval?

**Comment by PhiTux** [comment]
Hi, 
thanks very much to everybody involved for the translation! 
I'm not exactly sure right now, if I will "just" try to update dailyTxT within the next days and integrate you translation into this, or if I should start to rewrite the whole thing. 
There are several new ideas now in the issues-section, that I would like to address. But I would then combine this with a rewrite...
Well, let's see...

**Comment by PhiTux** [comment]
@Stokkie64 I'm not sure how I could remove you other approval. You mean the one right in front of the second approval, where you wrote this, right?

**Comment by Stokkie64** [comment]
I wouldn't stress about removing it but Thanks.

**Comment by PhiTux** [comment]
Just for your interest: I'm planing to start a rewrite of dailytxt within the next days. I'll try to address most of the open issues within this step. The dutch translation will be included into this (although there will surely be several new text-snippets which need to be translated...)

**Pull request merged**
